scala/src/com/google/idea/blaze/scala/run/Specs2Utils.java

```diff
@@ -98,4 +96,20 @@ public static String getTestFilter(ScTypeDefinition testClass, PsiElement testCa
     }
     return testClass.qualifiedName() + '#' + testName;
   }
+
+  @Nullable
+  private static ScInfixExpr getContainingExpr(PsiElement element, Predicate<PsiElement> predicate) {
+    while (element != null && !predicate.test(element)) {
+      element = PsiTreeUtil.getParentOfType(element, ScInfixExpr.class);
+    }
+    return (ScInfixExpr) element;
+  }
+
+  @Nullable
+  private static String getElementStaticText(PsiElement element) {
+    return TestConfigurationUtil.getStaticTestNameOrDefault(
+            element,
+            null,
```

Comment:
Ugh, they named the parameter "default". I think it's messing up the IDE.

Reply:
https://youtrack.jetbrains.com/issue/IDEA-182888

Reply:
Oh wow, so that's why the red squiggly :(
I upvoted the issue, let's hope it will be fixed before flying cars :)